src/synthesizer/particle/stars.py

```diff
@@ -565,8 +566,11 @@ def generate_lnu(
         )
 
         # Get the integrated spectra in grid units (erg / s / Hz)
-        lnu_particle = compute_integrated_sed(*args)
-
+        if shift=False:
+            lnu_particle = compute_integrated_sed(*args) 
+        else:
+            lnu_particle = compute_integrated_sed(*args) # need to modify this too?
```

Comment:
Yea, if we did do it this way then we need to call `compute_particle_seds_shifted` here (well in reality we want an integrated one here but we'll cross that bridge).